src/services/context/context.getter.service.ts

```diff
@@ -52,4 +53,55 @@ export class ContextGetterService extends GenericGetterService {
             Constants.oneMinute(),
         );
     }
+
+    async getNftsCountForUser(address: string): Promise<number> {
+        const cacheKey = this.getCacheKey('nftsCountForUser', address);
+        const cachedData = await this.cachingService.getRemote<number>(
+            cacheKey,
+        );
+        if (cachedData) {
+            return cachedData;
+        }
+        const count = await this.apiService.getNftsCountForUser(address);
+        await this.cachingService.setRemote(
```

Comment:
only remote?
you may use getOrSet

Reply:
I didn't want to have per instance caching with the local cache involved. All instances would use the cached value from remote cache